### DISCUSSION TITLE
edit web page

### DISCUSSION THREAD

**Discussion by ajeje989** [AzuraCast/AzuraCast#4490] [Q&A]
Hi .. nice audio control panel..

i have a question..

it is possible to modify the web part

like this https://prnt.sc/1p4896c

I installed everything via docker .. but I can't find anything in the var / azuracast folder

is there a guide on this?

**Comment by ajeje989** [comment]
no reply ?

**Comment by ivellios1988** [comment]
I believe it would be a better idea to embed the player on your website instead of modifying AzuraCast files.

Even if you find the localization of these files and modify them, they will likely be overwritten when you update your AzuraCast installation and all changes will be lost.

And if you have more than one station on your server then any changes in these files will apply to the public pages of all of your stations.

**Comment by ajeje989** [comment]
do you think it would be better to create my own panel and manage azuracast through api?

is there any guide?

**Comment by ivellios1988** [comment]
What would be the purpose of _creating a new panel and managing it through AzuraCast API_ when you can just create a simple website (i. e. using Wix), embed your player, and then customize literally everything?

And if you want new features to be added to AzuraCast then request them here: https://azuracast.featureupvote.com/
